lib/three_scale/domain_substitution.rb

```diff
@@ -12,7 +12,7 @@ def internal_host
   end
 
   # == Context
-  # In any debugging environment _development_ or _preview_,
+  # In any debugging environment. e.g. _development_,
```

Comment:
Is there any other debugging env?
```suggestion
  # In any debugging environment, i.e. _development_,
```